File: app/src/main/java/com/example/android/tictactoe/WinOrDrawDialog.java
package com.example.android.tictactoe;

import android.app.Dialog;
import android.os.Bundle;
import androidx.fragment.app.DialogFragment;
import androidx.appcompat.app.AlertDialog;
import android.view.ContextThemeWrapper;

public class WinOrDrawDialog extends DialogFragment {

    String mMessage;

    public WinOrDrawDialog(String message) {
        mMessage = message;
    }

    @Override
    public Dialog onCreateDialog(Bundle savedInstanceState) {
        AlertDialog.Builder builder = new AlertDialog.Builder(new ContextThemeWrapper(getActivity(),
                R.style.AlertDialogCustom));
        builder.setMessage(mMessage);

        // Create the AlertDialog
        return builder.create();
    }
}

File: app/src/main/java/com/zizohanto/android/tictactoe/WinOrDrawDialog.kt
package com.zizohanto.android.tictactoe

import android.app.Dialog
import android.os.Bundle
import android.view.ContextThemeWrapper
import androidx.annotation.StringRes
import androidx.appcompat.app.AlertDialog
import androidx.fragment.app.DialogFragment

class WinOrDrawDialog(@StringRes private val messageId: Int) : DialogFragment() {
    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
        return with(AlertDialog.Builder(ContextThemeWrapper(activity, R.style.AlertDialogCustom))) {
            setMessage(messageId)
            create()
        }
    }
}